test/src/e2e_vm_tests/test_programs/should_pass/language/type_layout/src/main.sw

```diff
@@ -0,0 +1,83 @@
+script;
```

Comment:
If we change this to `library` there will be no need for any `std` dependency in this test.
```suggestion
library;
```

Reply:
In this case, I actually can't, because the compiler only raises the type layout events when it is generating IR.